migrations/1602762361840-syncUserIndex.js

```diff
@@ -0,0 +1,16 @@
+const { info } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+require('../src/services/user/model');
+
+module.exports = {
```

Comment:
Please check if a migration without `down` method can be rolled back, just to be sure